mentoring/templates/html/ratingblock.html

```diff
@@ -6,38 +6,70 @@
   <div class="choices-list">
     <div class="choice">
       <div class="choice-result fa icon-2x"></div>
-      <label><input class="choice-selector" type="radio" name="{{ self.name }}" value="1"{% if self.student_choice == '1' %} checked{% endif %} />1</label>
-      <span class="low"> - {{ self.low }}</span>
-      <div class="choice-tips"></div>
+      <label class="choice-label">
```

Comment:
Thanks for the hint, that's much better.

When I tried to use `for value in [1, 2, 3, 4, 5]` I got `Error: 'for' statements should use the format 'for x in y': for value in [1, 2, 3, 4, 5]`, but `for value in '12345'` works fine.